frontend/src/views/WheresMyGene/common/types.ts

```diff
@@ -1,5 +1,6 @@
 import { CSSProperties } from "react";
 
+// This object holds a cell type and its genes.
```

Comment:
nit: Prefer multiline comment for typedoc support

Reply:
Ah great call. Thank you! Done!